templates/ros2_control/test_joint_trajectory_controller.launch.xml

```diff
@@ -0,0 +1,10 @@
+<launch>
```

Comment:
```suggestion
```suggestion
<!--
Copyright (c) 2024, Stogl Robotics Consulting UG (haftungsbeschränkt)

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.


Source of this file are templates in https://github.com/StoglRobotics/ros_team_workspace repository.

Authors: Riyan Jose, Dr. Denis
-->

<launch>
```